modules/core/src/main/java/org/apache/synapse/rest/RESTRequestHandler.java

```diff
@@ -149,4 +141,36 @@ private void apiProcessNonDefaultStrategy(MessageContext synCtx, API api) {
             api.process(synCtx);
         }
     }
+
+    public boolean identifyAPI(API api, MessageContext synCtx) {
+        Collection<API> apiSet = synCtx.getEnvironment().getSynapseConfiguration().getAPIs();
+        List<API> defaultStrategyApiSet = new ArrayList<API>(apiSet);
```

Comment:
Shall we get this list directly from the parent method?

Reply:
Also shall we make it private?

Reply:
Fixed